hubblestack/extmods/modules/pulsar.py

```diff
@@ -395,3 +395,44 @@ def _dict_update(dest, upd, recursive_update=True, merge_lists=False):
             for k in upd:
                 dest[k] = upd[k]
         return dest
+
+#TODO:change paths for top.pulsar and config yaml to 'salt://hubblestack_pulsar/top.file'
+
+def top(topfile='/root/myhubblefiles/topfile/top.pulsar',
+        debug=None):
+
+    results_pulsar = []
+
+    configs = get_top_data(topfile)
+
+    if not isinstance(configs, list):
+        configs = configs.split(',')
+
+    configs = ['/root/myhubblefiles/topfile/' + config + '.yaml'
+               for config in configs]
```

Comment:
Here, you'll want to prepend `salt://hubblestack_pulsar/` and append `.yaml` instead of the absolute path you're using.